privacy-deidentification/src/main/java/io/konig/privacy/deidentification/model/PersonKeys.java

```diff
@@ -12,7 +12,7 @@
 	private String pseudonym;
 	private List<String> email;
 	private List<Identity> identity;
-	
+	private String id;

```

Comment:
I don't think we should include this value.  The id can be derived from the pseudonym so there is no reason to capture it explicitly in PersonKeys.  Indeed, I notice that there is no code that actually calls the `getId` method.  Let's leave it out until there is a compelling use case to add it.